src/components/Organization/Dashboard/Votings.tsx

```diff
@@ -30,20 +31,22 @@ const VotingsList = ({ data, status }: VotingsProps) => {
   const { elections, pagination } = data
 
   return (
-    <Flex flexDirection='column' flexGrow={1} gap={5} height='full'>
-      {!!elections?.length ? (
-        <>
-          <ProcessesList processes={elections} />
-          <Flex mt='auto' justifyContent='end'>
-            <RoutedPagination pagination={pagination} />
-          </Flex>
-        </>
-      ) : !!status ? (
-        <NoResultsFiltering />
-      ) : (
-        <NoElections />
-      )}
-    </Flex>
+    <DashboardContents>
```

Comment:
This should not be set in the component, but in the element instead, since it's what's giving it the dashboard style.